mirrord-layer/src/lib.rs

```diff
@@ -117,6 +117,23 @@ fn is_nix_or_devbox() -> bool {
     }
 }
 
+/// Prevent mirrord from connecting to ports used by the intelliJ debugger
+pub(crate) fn intellij_debug_patch(port: Port) -> bool {
+    if let Ok(ports) = std::env::var("INTELLIJ_DEBUG_PATCH") {
+        // port range can be specified as "1234-1236" or just "1234"
+        let ports: Vec<u16> = ports.split('-').map(|p| p.parse().unwrap()).collect();
```

Comment:
Change `unwrap` here to `expect`, so you can give a nicer user error, like "Tried to parse a value that is not a number" or something along those lines.